db.json

```diff
@@ -5235,208 +5299,2001 @@
+        "ioc_serial": "21344",
+        "ioc_type": "Qmini",
+        "ioc_use_evr": "no",
+        "kwargs": {
+            "name": "{{name}}"
+        },
+        "last_edit": "Fri Sep 23 16:02:07 2022",
+        "lightpath": false,
+        "location_group": "3_lm1k2_hrm",
+        "name": "lm1k2_hrm_dp2_tf1_sp1",
+        "prefix": "LM1K2:HRM_DP2_TF1_SP1",
+        "stand": null,
+        "type": "pcdsdevices.happi.containers.Qmini",
+        "z": -1.0
+    },
+    "lm1k2_hrm_env_sens": {
+        "_id": "lm1k2_hrm_env_sens",
+        "active": true,
+        "args": [
+            "{{prefix}}"
+        ],
+        "beamline": "QRIX_MODS",
+        "creation": "Mon Sep 26 12:39:02 2022",
+        "device_class": "pcdsdevices.device_types.EnvironmentalMonitor",
+        "documentation": null,
+        "functional_group": "env sensors",
+        "ioc_arch": null,
+        "ioc_base": "LM1K2:HRM_BHC",
+        "ioc_engineer": "quarkahr",
+        "ioc_hutch": "LAS",
+        "ioc_ip": "bhc-las-qrixs-hrm-01",
+        "ioc_location": "qRIX MODS Table",
+        "ioc_name": "ioc-lm1k2-hrm-ek9000-01",
+        "ioc_release": "/cds/group/pcds/epics/ioc/common/ek9000/R1.3.1",
+        "ioc_type": "EK9000",
+        "kwargs": {
+            "name": "{{name}}"
+        },
+        "last_edit": "Mon Sep 26 16:43:09 2022",
+        "lightpath": false,
+        "location_group": "3_lm1k2_hrm",
+        "name": "lm1k2_hrm_env_sens",
+        "prefix": "LM1K2:HRM_BHC_AI_01",
+        "stand": null,
+        "type": "pcdsdevices.happi.containers.EnvironmentalMonitor",
+        "z": -1.0
+    },
+    "lm1k2_inj_dp1_tf1_nf1": {
+        "_id": "lm1k2_inj_dp1_tf1_nf1",
         "active": true,
         "args": [
             "{{prefix}}"
         ],
-        "beamline": "LAS",
-        "creation": "Wed Sep  8 13:13:08 2021",
+        "beamline": "QRIX_MODS",
+        "creation": "Tue Sep 20 17:28:24 2022",
         "device_class": "pcdsdevices.areadetector.detectors.LasBasler",
         "documentation": null,
         "functional_group": "cameras",
         "ioc_arch": "rhel7-x86_64",
         "ioc_cam_model": "Basler_acA1440-73gm",
-        "ioc_engineer": "tjohnson",
-        "ioc_evr_channel": "1",
-        "ioc_http_port": "7801",
+        "ioc_engineer": "quarkahr",
+        "ioc_evr_channel": "4",
+        "ioc_http_port": "7802",
         "ioc_hutch": "LAS",
-        "ioc_ip": "172.21.160.174",
-        "ioc_location": "ctl-las-lhn-04",
-        "ioc_name": "ioc-lhn-bay4-ff-01",
+        "ioc_ip": "TODO",
+        "ioc_location": "QRIX MODS Table",
+        "ioc_name": "ioc-lm1k2-inj-gige-nf-01",
         "ioc_net_if": "ETH0",
         "ioc_net_if_num": "1",
-        "ioc_release": "/cds/home/t/tjohnson/trunk/las/gigECam",
+        "ioc_release": "/reg/g/pcds/epics/ioc/common/gigECam/R3.3.0",
         "ioc_type": "BaslerGigE",
-        "ioc_use_evr": "172.21.160.174",
+        "ioc_use_evr": "yes",
```

Comment:
This is correct given the container, but I wonder why it's not a boolean value instead of the current string enforce type

Reply:
@slactjohnson?

Reply:
Looks like a copy-pasta error. 
@christina-pino and Aron were working on this. Aron is the `ioc_engineer` in this entry.

Reply:
NVM, I need to open my eyes. 
Yeah, this would probably be better as a boolean.